submarine-sdk/pysubmarine/submarine/cli/notebook/command.py

```diff
@@ -15,24 +15,132 @@
  under the License.
 """
 
+import json
+import time
+
 import click
+from rich.console import Console
+from rich.json import JSON as richJSON
+from rich.panel import Panel
+from rich.table import Table
+
+from submarine.cli.config.config import loadConfig
+from submarine.client.api.notebook_client import NotebookClient
+from submarine.client.exceptions import ApiException
+
+submarineCliConfig = loadConfig()
+if submarineCliConfig is None:
+    exit(1)
```

Comment:
It's better to raise an exception with error messages here.

Reply:
@pingsutw I have already raised the error message in `loadConfig`

https://github.com/apache/submarine/blob/efb4e7c578f9ee5188414b2140ab7b4a9de1ec92/submarine-sdk/pysubmarine/submarine/cli/config/config.py#L103-L114

Do I have to change the place to raise the error?